### ISSUE TITLE
[Dev] Restrict file uploads

### ISSUE THREAD

**Issue by thetif** [CMSgov/eAPD#2740]
There are currently no restrictions on file uploading. Add security checks on the file and implement other recommendations to fix this vulnerability

### This task is done when...
 - [ ] file types that can be uploaded are limited to only image files, such as .jpg/.gif
 - [ ] reject attempts to upload compressed archives such as ZIP files
 - [ ] verify uploaded file content, for instance, an uploaded JPEG should have the standard JPEG file headers
 - [ ] implement virus scanning on uploaded files
 - [ ] image files are not stored in a directory that is accessible by the user, preferably outside of the web root
 - [ ] restrict permissions on the image directory and the files uploaded to it that prevent execution, if possible deny access to everyone but the web application user

**Comment by cthulhuplus** [comment]
Jerome says we'll need to support PDF and Word formats "in the future."

**Comment by thetif** [comment]
I think we can cross that bridge when we get there, for now we can just restrict it to images.

**Comment by jeromeleecms** [comment]
That sounds good @thetif (and thanks for mentioning it @cthulhuplus). From some other conversations, it seemed like we needed to (1) restrict upload types (front end) and (2) validate the type when we actually stored it on S3 (backend). 

If we are already looking validation for storage on S3, I wanted to make sure we gave some consideration for a scalable solution for (2) if/when we need to expand the file types. For (1), it looks like TinyMCE already has the ability to limit the filetypes (I found some mention in the docs, but I'll defer to the Dev's about how best to implement).
